contracts/contracts/PoolFeeManagerProxy.sol

```diff
@@ -0,0 +1,131 @@
+        protectAddPool = true;
+    }
+
+    //-------------------------------------//
+    //---------- PoolManagerV4 ------------//
+    //-------------------------------------//
+    function setPoolManagerOperator(address _operator) external {
+        require(msg.sender == operator, "!auth");
+        IPoolManager(poolManager).setOperator(_operator);
+    }
+
+    function setPoolManagerProtectPool(bool _protectAddPool) external {
+        require(msg.sender == operator, "!auth");
+        IPoolManager(poolManager).setProtectPool(_protectAddPool);
+    }
+
+    function setOperator(address _operator) external {
+        require(msg.sender == operator, "!auth");
+        operator = _operator;
+    }
+  
+    /**
+     * @notice set if addPool is only callable by operator
+     */
+    function setProtectPool(bool _protectAddPool) external {
+        require(msg.sender == operator, "!auth");
+        protectAddPool = _protectAddPool;
+    }
+
+    /**
+     * @notice Add a new curve pool to the system. (default stash to v3)
+     */
+    function addPool(address _gauge) external returns(bool){
+        uint256 pid = IPools(booster).poolLength();
+        _addPool(_gauge);
+        (, , , address crvRewards, , ) = IPools(booster).poolInfo(pid);
+        _setRewardMultiplier(crvRewards, defaultRewardMultiplier);
+        return true;
+    }
+
+    function _addPool(address _gauge) internal returns(bool){
+        if(protectAddPool) {
+            require(msg.sender == operator, "!auth");
+        }
+        //get lp token from gauge
+        return IPoolManager(poolManager).addPool(_gauge);
+    }
+
+    function shutdownPool(uint256 _pid) external returns(bool){
+        require(msg.sender==operator, "!auth");
+
+        return IPoolManager(poolManager).shutdownPool(_pid);
+    }
+
+    //shutdown pool management and disallow new pools. change is immutable
+    function shutdownSystem() external {
+        require(msg.sender == operator, "!auth");
+        return IPoolManager(poolManager).shutdownSystem();
+    }
+
+    //----------------------------------//
+    //---------- Fee Manager------------//
+    //----------------------------------//
+    function _setRewardMultiplier(address _rewardContract, uint256 multiplier) internal {
+        IBoosterFees(booster).setRewardMultiplier(_rewardContract, multiplier);
+    }
+    function setDefaultRewardMultiplier(uint256 multiplier) external {
+        require(msg.sender == operator, "!auth");
+        require(multiplier <= REWARD_MULTIPLIER_DENOMINATOR * 2, "too high");
+
+        defaultRewardMultiplier = multiplier;
+        emit DefaultRewardMultiplierUpdated(multiplier);
+    }
+
+    function setFees(uint256 _lockFees, uint256 _stakerFees, uint256 _callerFees, uint256 _platform) external {
+        require(msg.sender == operator, "!auth");
+        IBoosterFees(booster).setFees(_lockFees, _stakerFees, _callerFees, _platform);
+    }
+    function setTreasury(address _treasury) external  {
```

Comment:
Have a little tidy up of this code in general, spacing between functions etc should try and stay consistent. Surprised the formatter didn't fix this tbh

Reply:
Done sir